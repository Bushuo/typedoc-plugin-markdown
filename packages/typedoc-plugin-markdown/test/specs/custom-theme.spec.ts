import * as path from 'path';
import { TestApp } from '../test-app';

describe(`CustomTheme:`, () => {
  let testApp: TestApp;

  beforeAll(async () => {
    testApp = new TestApp(['breadcrumbs.ts']);
    const customThemePath = path.resolve(
      __dirname,
      '..',
      'stubs',
      'custom-theme',
    );
    await testApp.bootstrap({
      theme: customThemePath,
    });
  });

  test(`should load custom theme`, () => {
    expect(testApp.theme.basePath).toMatchSnapshot();
  });
});
