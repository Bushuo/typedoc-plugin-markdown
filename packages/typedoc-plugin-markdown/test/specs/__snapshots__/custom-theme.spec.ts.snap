// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTheme: should load custom theme 1`] = `"/Users/tomgrey/Projects/typedoc-plugin-markdown/packages/typedoc-plugin-markdown/test/stubs/custom-theme"`;
